apps/frontend/src/app/libs/sidenav/sidenav.component.html

```diff
@@ -20,57 +20,83 @@
+        <a
+          mat-list-item
+          *ngIf="isLoggedIn"
+          routerLink="sessions"
+          routerLinkActive="active-link"
+        >
           <mat-icon>watch_later</mat-icon>
           <span>{{ 'SESSIONS.SESSION' | translate }}</span></a
         >
-        <a mat-list-item
-           *ngIf="isLoggedIn"
-           routerLink="rooms" routerLinkActive="active-link">
+        <a
+          mat-list-item
+          *ngIf="isLoggedIn"
+          routerLink="rooms"
+          routerLinkActive="active-link"
+        >
           <mat-icon>home</mat-icon>
           <span>{{ 'ROOMS.ROOMS' | translate }}</span></a
         >
-        <a mat-list-item
-           *ngIf="isLoggedIn"
-           routerLink="profile" routerLinkActive="active-link">
+        <a
+          mat-list-item
+          *ngIf="isLoggedIn"
+          routerLink="profile"
+          routerLinkActive="active-link"
+        >
           <mat-icon>person</mat-icon>
           <span>{{ 'PROFILES.PROFILE' | translate }}</span></a
         >
-        <a mat-list-item
-           *ngIf="!isLoggedIn"
-           [class.active] = "isSignup"
-           (click)='changeAuthState(true)'
-           routerLink="auth"
-           >
+        <a
+          mat-list-item
+          *ngIf="!isLoggedIn"
+          [class.active]="isSignup"
+          (click)="changeAuthState(true)"
+          routerLink="auth"
+        >
           <mat-icon>home</mat-icon>
           <span>{{ 'REGISTER' | translate }}</span></a
         >
-        <a mat-list-item
-           *ngIf="!isLoggedIn "
-           [class.active] = "!isSignup"
-           (click)='changeAuthState(false)'
-           routerLink="auth">
+        <a
+          mat-list-item
+          *ngIf="!isLoggedIn"
+          [class.active]="!isSignup"
+          (click)="changeAuthState(false)"
+          routerLink="auth"
+        >
           <mat-icon>home</mat-icon>
           <span>{{ 'LOGIN' | translate }}</span></a
         >
-<!--        <a-->
-<!--          mat-list-item-->
-<!--          *ngIf="adminService.isAdmin"-->
-<!--          routerLink="./admin"-->
-<!--          routerLinkActive="active-link"-->
-<!--        >-->
-<!--          <mat-icon>person</mat-icon>-->
-<!--          <span>{{ 'onlyAdmin' | translate }}</span></a-->
-<!--        >-->
+        <!--        <a-->
+        <!--          mat-list-item-->
+        <!--          *ngIf="adminService.isAdmin"-->
+        <!--          routerLink="./admin"-->
+        <!--          routerLinkActive="active-link"-->
+        <!--        >-->
+        <!--          <mat-icon>person</mat-icon>-->
+        <!--          <span>{{ 'onlyAdmin' | translate }}</span></a-->
+        <!--        >-->
       </mat-nav-list>
+
+      <h3
+        mat-list-item
+        class="error-color pointer"
+        *ngIf="isLoggedIn"
+        (click)="handleLogout()"
+      >
+        <mat-icon style="vertical-align: middle">logout</mat-icon>
```

Comment:
Enthält immer noch inline-style.
Oder stört dich das an der Stelle nicht, @bilal0710 ?

Reply:
doch, es sollten alle Stiländerungen in er Scss gemacht werden :)